auslib/web/views/client.py

```diff
@@ -52,7 +52,34 @@ def get(self, **url):
         release, update_type = AUS.evaluateRules(query)
         # passing {},None returns empty xml
         if release:
-            xml = release.createXML(query, update_type, app.config["WHITELISTED_DOMAINS"], app.config["SPECIAL_FORCE_HOSTS"])
+            response_products = release.getResponseProducts()
```

Comment:
I think this block could be improved if a list of response blobs were built up first, and then the XML processed.
response_blobs = []
response_products = release.getResponeProducts()
if response_products:
   blob = AUS.evaluateRules(...)
   release_blobs.append(blob)
else:
   response_blobs.append(release)

With the new response_blobs array, you could have a single piece of code that handles SuperBlob and non-SuperBlob responses with the new get*XML() methods, and we could kill createXML entirely.

What do you think about that?